dataland-frontend/src/components/forms/parts/elements/basic/UploadDocumentsForm.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <!-- Select company reports -->
-  <div class="uploaded-files" style="padding: 0; margin-left: 0">
+  <div id="uploadReports" class="uploaded-files" style="padding: 0; margin-left: 0">
```

Comment:
What is this ID used for?

Reply:
I think in validationUtils it is used to determine unused reports